etc/docker/nb2kg/Dockerfile

```diff
@@ -1,24 +1,22 @@
 # Copyright (c) Jupyter Development Team.
 # Distributed under the terms of the Modified BSD License.
 
-FROM jupyter/minimal-notebook
+FROM jupyterhub/k8s-singleuser-sample:0.7.0
 
 # pip packages
 RUN pip install --upgrade pip
 RUN pip install setuptools --ignore-installed --upgrade
 
-# Do the pip installs as the unprivileged notebook user
-USER jovyan
+ENV NBGITPULLER_APP=lab
 
-# Install Lab and NB2KG. Enable NB2KG extension.  Lab extension is optionally enabled at runtime.
-RUN pip install --upgrade jupyter_contrib_nbextensions jupyterlab nb2kg && \
-    jupyter contrib nbextension install --sys-prefix && \
-    jupyter serverextension enable --py nb2kg --sys-prefix
-
-ADD start-nb2kg.sh /usr/local/share/jupyter/
+# Install Lab and NB2KG. Enable NB2KG extension. Lab is enabled by default
+RUN pip install --upgrade jupyterlab nb2kg && \
+    jupyter serverextension enable --py nb2kg --sys-prefix && \
+    jupyter serverextension enable --py jupyterlab --sys-prefix
 
```

Comment:
Looks like this installs the lab extension unconditionally.  Previously, lab would only be enabled if the command line of the image indicated that it should be enabled.  I think we should preserve the ability to use this image in Noteb ook or Lab if possible.

Reply:
So, currently, if you go to http://host:8888 it will use the default notebook ui, otherwise, if you use http://host:8888/lab it will use jupyterlab ui.